src/storybook/stand-alone-documentaion/welcome/ability-description/ability-description.jsx

```diff
@@ -0,0 +1,16 @@
+import { BEMClass } from "../../../../helpers/bem-helper";
+import Link from "../../../../components/Link/Link";
+import "./ability-description.scss";
+
+const BASE_CLASS = "monday-storybook-ability-description";
+const bemHelper = BEMClass(BASE_CLASS);
+export const AbilityDescription = ({ title, children, imageSrc, linkHref }) => {
+  return (
+    <section className={BASE_CLASS}>
```

Comment:
here I am not using the information box because this section look different than the information box design/